tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist =
+    py{27,33,34,35,36,37}
+
+[testenv]
+setenv =
+    PYTHONDONTWRITEBYTECODE=1
```

Comment:
I think I copied it from other projcets and have kept it. Prevents `.pyc` files being used when you switch branches (???)